frappe/website/js/web_form.js

```diff
@@ -36,6 +36,11 @@ frappe.ready(function() {
 	setTimeout(() => {
 		$('body').css('display', 'block');
 
+		// remove footer save button if form height is grater less than window height
+		if(!($('.webform-wrapper').height() > window.innerHeight)){
```

Comment:
Why not...
```suggestion
		if($('.webform-wrapper').height() < window.innerHeight) {
```
Saves a negation operation...

Reply:
oh why i have done this.

Reply:
done